client/container_copy.go

```diff
@@ -42,7 +42,7 @@ func (cli *Client) CopyToContainer(ctx context.Context, container, path string,
 		query.Set("copyUIDGID", "true")
 	}
 
-	apiPath := fmt.Sprintf("/containers/%s/archive", container)
+	apiPath := "/containers/" + container + "/archive"
```

Comment:
Wondering if we should use `path.Join()` for these (although they're of course URL's, so probably won't make much of a difference)